src/app/config/schema.ts

```diff
@@ -238,6 +241,62 @@ export const optionalVarsSchema: Schema<IOptionalVarsSchema> = {
       env: 'CUSTOM_CLOUDWATCH_LOG_GROUP',
     },
   },
+  sgid: {
```

Comment:
might it be better to put these in a separate file like in `spcp-myinfo.config`? might be easier to keep the slices of config more manageable

Reply:
I don't mind, let me rework this